CREDITS.md

```diff
@@ -236,6 +236,7 @@ Contributors to the Firecracker repository:
 - Stefan Nita <32079871+stefannita01@users.noreply.github.com>
 - StemCll <lydjotj6f@mozmail.com>
 - Sudan Landge <sudanl@amazon.com>
+- sudanl0 <119602619+sudanl0@users.noreply.github.com>
```

Comment:
We can backport df56fef5568aa1c4a91d442971ea1dc970df6ce9 or fix it manually

Reply:
thanks, done